src/test/regress/sql/multi_modifying_xacts.sql

```diff
@@ -165,7 +165,7 @@ SELECT name FROM labs WHERE id = 10;
 INSERT INTO labs VALUES (6, 'Bell Labs');
 COMMIT;
 
--- but a double-copy isn't allowed (the first will persist)
+-- double-copy can also allowed
```

Comment:
I don't really know what "double-copy" is ;). "Two consecutive COPYs in a transaction are allowed"?